### PR TITLE
Feat/handle colored lights

### PR DESCRIPTION
**Pull request by mathieurobert1**
Handle colored lights.
Here an example with two lights whithout color:
image
And with a red light and a purple one:
image
More informations about config file : #74

### CODE REVIEW

File: src/Plugins/Lights/Ambient/Ambient.cpp [hunk 1]
```diff
@@ -52,3 +54,8 @@ Color Light::Ambient::computeColor(Math::Vector3D primitiveNormal, const Math::P
     color *= _multiplier;
     return color;
 }
+
+Color Light::Ambient::getColor(void) const
+{
+    return _color;
+}
```
Comment (AxelF44) [inline src/Plugins/Lights/Ambient/Ambient.cpp:61]:
```suggestion
}

```

File: src/Plugins/Lights/Directional/Directional.cpp [hunk 2]
```diff
@@ -87,5 +89,16 @@ Color Light::Directional::computeColor(Math::Vector3D primitiveNormal, const Mat
...
+
+void Light::Directional::setColor(const Color& rgb)
+{
+    _color = rgb;
+}
+
+Color Light::Directional::getColor(void) const
+{
+    return _color;
+}
```
Comment (AxelF44) [inline src/Plugins/Lights/Directional/Directional.cpp:104]:
```suggestion
}

```

**Review by AxelF44 — CHANGES_REQUESTED** [review]

**Review by theophile-jr — CHANGES_REQUESTED** [review]
fix current requests changes please.

File: tests/colorLight.cfg
Comment (Njord201) [inline tests/colorLight.cfg:1]: Move this files in tests/files_examples.

**Review by Njord201 — CHANGES_REQUESTED** [review]
One request but good job !

**Review by theophile-jr — APPROVED** [review]

**Review by AxelF44 — APPROVED** [review]

**Pull request merged**